Translated Conditional and List. (#7)
1950Labs/docs-next: src/guide/list.md

# Renderizado de Lista

> ## Mapeando una matriz a Elementos con `v-for`

Reviewer: What do you think about this?

```suggestion
## Mapeando un Arreglo a Elementos con `v-for`
```

Author: Thanks, @migsalazar! I forgot we're using `arreglo` instead of `matriz`.

> Podemos usar la directiva `v-for` para renderizar una lista de elementos basada en una matriz. La directiva `v-for` requiere una sintaxis especial en forma de `item in items`, donde los `items` son la matriz de datos de origen y el `item` es un **alias** para el elemento de matriz que se está iterando:

Reviewer: Same here -as my previous comment-

```suggestion
Podemos usar la directiva `v-for` para renderizar una lista de elementos basada en un arreglo. La directiva `v-for` requiere una sintaxis especial en forma de `item in items`, donde los `items` son el arreglo de datos de origen y el `item` es un **alias** para el elemento de matriz que se está iterando:
```

Para proporcionar a Vue una sugerencia para que pueda rastrear la identidad de cada nodo y, por lo tanto, reutilizar y reordenar los elementos existentes, debe proporcionar un atributo `key` único para cada elemento:

```html
<div v-for="item in items" :key="item.id">
  <!-- contenido -->
</div>
```

Se recomienda proporcionar un atributo `key` con `v-for` siempre que sea posible, a menos que el contenido DOM iterado sea simple, o esté confiando intencionalmente en el comportamiento predeterminado para obtener ganancias en el rendimiento.

Como identificar nodos es un mecanismo genérico de Vue, `key` también tiene otros usos que no están específicamente vinculados a `v-for`, como veremos más adelante en la guía.

:::tip Nota
No utilice valores no primitivos como objetos y vectores como clave para `v-for`. Utilice _string_ o valores numéricos en su lugar.
:::

Para ver el uso en detalle del atributo `key`, por favor vea la [documentación de la API para `key`](../api/special-attributes.html#key).

> ## Detección del cambios en Array

Reviewer 2: ```suggestion
## Detección del cambios en Arreglos
```

- `sort()`
- `reverse()`

Puede abrir la consola y probar con la matriz de `items` de los ejemplos anteriores llamando a sus métodos de mutación. Por ejemplo: `example1.items.push({ message: 'Baz' })`.

> ### Mutando un Array

Reviewer 2: ```suggestion
### Mutando un Arreglo
```
